src/sourceMaps/sourceMap.ts

```diff
@@ -132,25 +133,14 @@ export class SourceMap {
         // sourceMap.sources are initially relative paths, file:/// urls, made-up urls like webpack:///./app.js, or paths that start with /.
         // resolve them to file:/// urls, using computedSourceRoot, to be simpler and unambiguous, since
         // it needs to look them up later in exactly the same format.
-        const normalizedSources = sourceMap.sources.map(sourcePath => {
-            if (isDefined(sourceMapPathOverrides)) {
-                const fullSourceEntry = sourceMapUtils.getFullSourceEntry(sourceMap.sourceRoot, sourcePath);
-                const mappedFullSourceEntry = sourceMapUtils.applySourceMapPathOverrides(fullSourceEntry.textRepresentation, sourceMapPathOverrides, isVSClient);
-                if (fullSourceEntry.textRepresentation !== mappedFullSourceEntry) {
-                    return mappedFullSourceEntry; // If we found a path override that applies, return the result of applying it
-                }
-            }
-
-            if (sourcePath.startsWith('file://')) {
-                // strip file://
-                return new LocalFileURL(sourcePath).filePathRepresentation;
-            }
-
-            if (!path.isAbsolute(sourcePath)) {
-                // Overrides not applied, use the computed sourceRoot
-                return path.resolve(computedSourceRoot, sourcePath);
-            } else {
-                return sourcePath;
+        const normalizedSources: string[] = [];
```

Comment:
Is it a problem that the indexes of the sources will change at this point if you just skip ones that fail? It looks like sourceMap.sources is referenced after this point and I can't tell if it's a problem that `sourceMap.sources.length !== normalizedSources.length`.

Reply:
Yes, that will most likely be an issue. Good catch :)

Now we add the unormalized source when things fail:
                // We add the unnormalized source when things fail. Processing this source will most likely fail in other places of the code.
                // We need to insert something to preserve the indexes of the sources matching the ones for the mappings, etc...
                normalizedSources.push(`${sourcePath}`);